src/components/molecules/card/card.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<div class="card-body">
+  <div class="box-image">
```

Comment:
```suggestion
  <div class="image-wrapper">
```
変化がなかったのはクラス名がcssとhtmlで違うからかなと思います。このように修正してみてください！